cocos/2d/CCGrid.h

```diff
@@ -186,11 +227,19 @@ class CC_DLL Grid3D : public GridBase
     virtual void calculateVertexPoints() override;
     /**@}*/
     
-    /**@{
-     Getter and Setter for depth test state when blit.
+    /**@{@~english
+     * Set the depth test state when blit.
+     * @~chinese 
+     * 设置位块传送时的深度测试状态。
```

Comment:
The translation of `blit` seems a little strange. Could we use the original `blit` instead?